alert-common/src/main/java/com/synopsys/integration/alert/common/action/api/AbstractResourceActions.java

```diff
@@ -65,6 +69,7 @@ public AbstractResourceActions(DescriptorKey descriptorKey, ConfigContextEnum co
     @Override
     public final ActionResponse<T> create(T resource) {
         if (!authorizationManager.hasCreatePermission(context.name(), descriptorKey.getUniversalKey())) {
+            logger.error("Create action is forbidden. This user is not authorized to perform this action.");
```

Comment:
If we want to log something every time a permission check is failed, then perhaps we should add this logging to the authorization manager instead of the Actions.

Reply:
That would probably allow us to log the username as well.

Reply:
I can try to add some logging in AuthorizationManager, but would we still want a log here? The authorization manager wouldn't know what action was tried it would only know what type of permission was tested.